src/popup/components/Main.vue

```diff
@@ -472,19 +490,41 @@ export default {
       });
     },
     async deleteAlias(index) {
-      this.aliasArray[index].loading = true;
+      const alias = this.aliasArray[index];
+      alias.moreOptions.loading = true;
       const res = await callAPI(
         API_ROUTE.DELETE_ALIAS,
         {
-          alias_id: this.aliasArray[index].id,
+          alias_id: alias.id,
         },
         {},
         API_ON_ERR.TOAST
       );
       if (res) {
         this.aliasArray.splice(index, 1);
       } else {
-        this.aliasArray[index].loading = false;
+        alias.moreOptions.loading = false;
+      }
+    },
+    async handleClickSave(index) {
+      const alias = this.aliasArray[index];
+      alias.moreOptions.loading = true;
+      const res = await callAPI(
+        API_ROUTE.EDIT_ALIAS,
+        {
+          alias_id: alias.id,
+        },
+        {
+          note: alias.moreOptions.note,
+        },
+        API_ON_ERR.TOAST
+      );
+      if (res) {
+        Utils.showSuccess("Updated alias");
+        alias.note = alias.moreOptions.note;
+        alias.moreOptions = null;
```

Comment:
could we leave the "more" section opened when the note is updated? So if user makes a mistake/typo, they can fix quickly fix it.